sdk/appplatform/Azure.ResourceManager.AppPlatform/src/Generated/Models/DiagnosticParameters.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+namespace Azure.ResourceManager.AppPlatform.Models
+{
+    /// <summary> Diagnostic parameters of diagnostic operations. </summary>
+    public partial class DiagnosticParameters
```

Comment:
Maybe rename this model to `DiagnosticContent` as it is used as the input parameter.